src/expressjs/app.js

```diff
@@ -0,0 +1,21 @@
+import express from 'express';
+import bodyParser from 'body-parser';
+import { cookieParser } from './middlewares/cookieParser';
+import { queryParser } from './middlewares/queryParser';
+import userRouter from './routes/users/'
+import productRouter from './routes/products/'
+
+const app = express();
+
+app.use(cookieParser());
+app.use(queryParser());
+
+app.use(bodyParser.json());
```

Comment:
AFAIK this is not needed anymore - express brought back an ability to parse body from the box. Please check [this link](https://expressjs.com/ru/4x/api.html#express.json)

Reply:
Thanks, updated code.